### PR TITLE
Add bean mapping category.

### PR DESCRIPTION
**Pull request by f-lopes**
Add Dozer and MapStruct to bean mapping category.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +73,12 @@ A curated list of awesome Java frameworks, libraries and software.
 * [JavaServer Faces](https://javaserverfaces.java.net/) - Oracle's open-source implementation of the JSF standard, Mojarra.
 * [JavaServer Pages](https://jsp.java.net/) - Common templating for websites with custom tag libraries.
 
+## Bean Mapping
+
+*Frameworks that ease bean mapping.*
+
+* [Dozer](http://dozer.sourceforge.net/) - Mapper that copies data from one object to another, using annotations, API or XML configuration.
```
Comment (akullpp) [inline README.md]: Please use the GitHub URL.

File: README.md [hunk 2]
```diff
@@ -72,6 +73,12 @@ A curated list of awesome Java frameworks, libraries and software.
 * [JavaServer Faces](https://javaserverfaces.java.net/) - Oracle's open-source implementation of the JSF standard, Mojarra.
 * [JavaServer Pages](https://jsp.java.net/) - Common templating for websites with custom tag libraries.
 
+## Bean Mapping
```
Comment (akullpp) [inline README.md]: You could also add https://github.com/xebia-france/selma

**Comment by wjam** [comment]
There's also [Orika](https://github.com/orika-mapper/orika) and [ModelMapper](http://modelmapper.org/), although I haven't had much experience with ModelMapper.

**Comment by f-lopes** [comment]
I didn't add Selma, Orika or ModelMapper as I have never used them.

I will add these tools anyway.

Thanks akullp & wjam.

**Pull request merged**